tests/foreman/ui/test_documentation_links.py

```diff
@@ -0,0 +1,92 @@
+"""Test module for verifying Documentation links
+
+:Requirement: Branding
+
+:CaseAutomation: Automated
+
+:CaseComponent: Branding
+
+:Team: Platform
+
+:CaseImportance: High
+
+"""
+
+from collections import defaultdict
+
+import pytest
+import requests
+
+from robottelo.config import settings
+from robottelo.logging import logger
+
+
+@pytest.mark.e2e
+@pytest.mark.skipif(
+    (settings.server.version.release.split('.')[0:2] in settings.robottelo.sat_non_ga_versions),
+    reason="The test don't yet support verifying documentation links for non GA'ed Satellite release.",
+)
+def test_positive_documentation_links(target_sat):
+    """Verify that Satellite documentation links are working.
+        Note: At the moment, the test doesn't support verifying links hidden behind a button.
+        Currently, the only such link is on RH Cloud > Inventory Upload page.
+
+    :id: 5e6cba22-896a-4d86-95b4-87d0cc2f1cb6
+
+    :Steps:
+
+        1. Gather documentation links present on various Satellite pages.
+        2. Verify the links are working (returns 200).
+
+    :expectedresults: All the Documentation links present on Satellite are working
+    """
+    pages = [
+        'about',
+        'settings',
+        'bookmark',
+        'role',
+        'ldapauthentication',
+        'cloudinventory',
+        'ansiblevariables',
+        'ansibleroles',
+        'discoveryrule',
+        'global_parameter',
+        'oscapreport',
+        'oscappolicy',
+        'oscapcontent',
+        'jobtemplate',
+        'provisioningtemplate',
+        'partitiontable',
+        'operatingsystem',
+        'host',
+        'discoveredhosts',
+        'reporttemplate',
+        'configreport',
+        'jobinvocation',
+        'audit',
+        'factvalue',
+        'dashboard',
```

Comment:
slightly concerned about keeping this list up to date

Reply:
Can't think of better approach atm:/